saleor/checkout/migrations/0023_merge_20200109_1402.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 2.2.9 on 2020-01-09 20:02
```

Comment:
Please rebase to the newest master. Checkout Permission is already added there. So, you don't need to modify DB